rekuho/src/index.test.ts

```diff
@@ -0,0 +1,7 @@
+import { describe, it, expect } from "vitest"
+
+describe("sum test", () => {
+  it("adds 1 + 2 to equal 3", () => {
+    expect(1 + 2).toBe(3)
+  })
+})
```

Comment:
Why do we have this again?

Reply:
Because it was here when my `npm` helper created the project, will change in the future. Hopefully.